data/test/scenarios/test_event_names_and_order.cfg

```diff
@@ -1,3 +1,12 @@
+#####
+# API(s) being tested: [event]
+##
+# Actions:
+# Define an event using a variable in its name attribute.
+##
+# Expected end state:
+# The variable is interpolated and the event executes since it is defined before the side 2 turn 1 event.
```

Comment:
This documents a test that should exist, but it's not quite what this test does. The rest of this review comment expanded to become PR #7160.

Reply:
Done.